sdks/python/apache_beam/dataframe/schemas_test.py

```diff
@@ -36,6 +36,8 @@
 from apache_beam.testing.test_pipeline import TestPipeline
 from apache_beam.testing.util import assert_that
 from apache_beam.testing.util import equal_to
+from apache_beam.typehints import typehints
+from apache_beam.typehints.native_type_compatibility import _match_is_named_tuple
```

Comment:
nit: remove the leading underscore since `_match_is_named_tuple` is no longer private.

Reply:
Done.